PH-1/app/ingestion/router.py

```
@@ -16,14 +18,14 @@
     'image': 20 * 1024 * 1024, 'video': 500 * 1024 * 1024,
 }
 
-def detect_file_type(filename: str) -> Union[str, Literal['unsupported']]:
+async def detect_file_type(filename: str) -> Union[str, Literal['unsupported']]:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)